packages/schematics/angular/web-worker/index.ts

```diff
@@ -169,11 +133,11 @@ export default function (options: WebWorkerOptions): Rule {
     const parsedPath = parseName(options.path, options.name);
     options.name = parsedPath.name;
     options.path = parsedPath.path;
-    const root = project.root || project.sourceRoot || '';
+    const root = project.root || '';
 
     const needWebWorkerConfig = !projectTargetOptions.webWorkerTsConfig;
     if (needWebWorkerConfig) {
-      const workerConfigPath = `${root.endsWith('/') ? root : root + '/'}tsconfig.worker.json`;
+      const workerConfigPath = join(normalize(root), 'tsconfig.worker.json');
```

Comment:
Won't this result in a normalized absolute path? If so, is that intended?
(Or is `project.root` relative?)

Reply:
Project root is relative 😊